### PR TITLE
avtar changes

### PR DESCRIPTION
**Pull request by rahulneal**

### CODE REVIEW

**Comment by pratik-kumar-sahu** [comment]
For better code formatting you can use VS Code extension "Prettier". It will take care of the uneven spaces in your code.

File: docs.html [hunk 1]
```diff
@@ -87,9 +87,45 @@ <h6>A navigation bar is a section of a graphical user interface intended to aid
...
...0%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cimg%2520src%253D%2522.%252Fcomponent_image%252Fvalo3.png%2522%2520alt%253D%2522avatar-3%2522%2520width%253D%2522112px%2522%2520height%253D%2522112px%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Ful%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E"
```
Comment (pratik-kumar-sahu) [inline docs.html:125]: If possible, you can also use tinyurl to make this url even shorter.

File: component_styles/badge.css [hunk 2]
```diff
@@ -0,0 +1,75 @@
+.badge-showcase{
+    display: flex;
+    display: flex;
```
Comment (shashikantmaurya0007) [inline component_styles/badge.css:3]: Here you have used two display

**Review by shashikantmaurya0007 — APPROVED** [review]
Looks good to me

File: docs.html (hunk 1 above)
Comment (rahulneal) [reply docs.html:125]: sure

File: component_styles/badge.css (hunk 2 above)
Comment (rahulneal) [reply component_styles/badge.css:3]: ok

**Pull request merged**